### ISSUE TITLE
Cleaned-up v1.18

### ISSUE THREAD

**Issue by comradau** [pmos69/dso203_gcc#8]
Hi Pedro,

I have uploaded the 'cleaned up' version 1.18 of the source. I have not been able to work out how to issue a pull request to you.

Peter

**Comment by pmos69** [comment]
Never did it myself, but it should be with the button at the very top of the screen, on the right (Pull request)...

**Comment by pmos69** [comment]
I've now updated to 1.20, but you can merge the differences...

**Comment by pmos69** [comment]
I don't think you actually pushed your modifications to your repository, because I can still only see the original forked files: https://github.com/comradau/dso203_gcc

If you update your repository, I think I can try and merge your changes in mine.

**Comment by pmos69** [comment]
(that is if that's your repository :) - It might not be)

**Comment by comradau** [comment]
Hi Pedro,

I'll have another go...

Peter Philippa

-----Original Message-----

**Comment by comradau** [comment]
Hi Pedro,

From what I can understand of the instructions on github, I need to create a tag to do a submission, am I right?

If so, I won't be able to do a submission because I am on a Win7 machine, and from the instructions on github it seems that a UNIX (or Linux) machine is needed for the command line entries needed to do a submission... again, is this correct.

Peter Philippa
-----Original Message-----

**Comment by pmos69** [comment]
Sorry for delay (that's an understatement).

You can use a Git Gui, like http://windows.github.com/ on your windows machine.
